applications/DamApplication/custom_strategies/schemes/bossak_displacement_smoothing_scheme.hpp

```diff
@@ -57,6 +57,21 @@ class BossakDisplacementSmoothingScheme : public ResidualBasedBossakDisplacement
         r_model_part.GetProcessInfo()[RAYLEIGH_ALPHA] = mRayleighAlpha;
         r_model_part.GetProcessInfo()[RAYLEIGH_BETA] = mRayleighBeta;
 
+        const int NNodes = static_cast<int>(r_model_part.Nodes().size());
+        ModelPart::NodesContainerType::iterator node_begin = r_model_part.NodesBegin();
+
+        // Initialize INITIAL_STRESS_TENSOR
+        #pragma omp parallel for
+        for(int i = 0; i < NNodes; i++)
+        {
+            ModelPart::NodesContainerType::iterator itNode = node_begin + i;
+
+            Matrix& rInitialStress = itNode->FastGetSolutionStepValue(INITIAL_STRESS_TENSOR);
```

Comment:
```suggestion
            auto& r_initial_stress = itNode->FastGetSolutionStepValue(INITIAL_STRESS_TENSOR);
```
Also note the `snake_case` for local variables as defined in our [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide).